swarm/pss/api.go

```diff
@@ -157,14 +157,23 @@ func (pssapi *API) StringToTopic(topicstring string) (Topic, error) {
 }
 
 func (pssapi *API) SendAsym(pubkeyhex string, topic Topic, msg hexutil.Bytes) error {
+	if !checkMsg(msg) {
+		return errors.New("invalid message")
```

Comment:
Probably a good idea to export this to `ErrInvalidMessage` in case clients want to handle it.